outreach-communication.qmd

```diff
@@ -0,0 +1,251 @@
+---
+title: "Communications and Outreach"
+---
+
+## Documents and Websites to Update regularly
+
+-   [CCT Data Science website](https://datascience.cct.arizona.edu/)
+
+-   [CCT Service Catalog](https://docs.google.com/document/d/1LIxkktiP219_Xd24vXpgA1Qz7j7WcuRORtN26p2Qujw/edit)
+
+## Where to announce events:
+
+-   Our [email list](#emma) (try to consolidate events to quarterly emails)
+
+-   [UA Master Calendar](https://news.arizona.edu/calendar)
+
+-   [Data Science Institute events page](https://datascience.arizona.edu/events)
+
+-   Experiment Station Tues morning notes: send announcement to [tmn\@cals.arizona.edu](mailto:tmn@cals.arizona.edu)
+
+-   [CALS Bulletin](https://compass.arizona.edu/bulletin)
+
+-   CALS Grad Student List Serv: send announcement to Dr. Jia Hu [jiahu\@arizona.edu](mailto:jiahu@arizona.edu)
+
+-   Individual CALS departments listservs as appropriate:
+
+    <!--# TODO: these likely need updating -->
+
+    -   [ruthholladay\@arizona.edu](mailto:ruthholladay@arizona.edu) Ruth Holladay for SNRE weekly digest
+
+    -   [jbee\@arizona.edu](mailto:jbee@arizona.edu) Jessica Bee for FCSC/Human Ecology grad student listserv
+
+    -   \_\_\_ for Plant sciences grad student listserv
+
+    -   [jabrilithomas\@arizona.edu](mailto:jabrilithomas@arizona.edu) Jabril Thomas for ACBS
+
+    -   [davaj\@arizona.edu](mailto:davaj@arizona.edu) Dava Jondall for Biosystems Engineering
+
+    -   [danielleb\@arizona.edu](mailto:danielleb@arizona.edu) Danielle Pilar Buhrow for AREC
+
+    -   [nmcisaac\@arizona.edu](mailto:nmcisaac@arizona.edu) Naomi McIsaac for Environmental Science
+
+    -   [prupert\@arizona.edu](mailto:prupert@arizona.edu) Peggy Rupert for Nutritional Sciences and Wellness
+
+    -   [pnielsen322\@arizona.edu](mailto:pnielsen322@arizona.edu) Paula Nielsen for Entomology
+
+    -   [thodges\@arizona.edu](mailto:thodges@arizona.edu) Tanya Hodges for AETI (Yuma)
+
+-   Data Science Institute ["Data Drip"](https://bcf.arl.arizona.edu/news) at [Rii-datascienceinstitute\@arizona.edu](mailto:Rii-datascienceinstitute@arizona.edu) <!--# this URL is broken.  What is "Data Drip" and does it exist still? -->
```

Comment:
I checked out the `https://bcf.arl.arizona.edu/news` link (aka: data drip news) and it looks good. Is that the URL that was broken? **Data Drip** appears to be what they call the monthly news letter

Reply:
Huh, might have been a temporary issue